src/main/resources/public/js/src/launches/common/InfoPanelView.js

```diff
@@ -134,6 +134,7 @@ define(function (require, exports, module) {
         },
         update: function () {
             this.loadDefectTooltip();
+            this.loadExecutionsTooltip();
```

Comment:
Create a plugin without removing the old one.
Create a view for the tooltip in this file, and put it in the tooltip. Then just change the view after changing the model.